test/functional/test-viewport.js

```diff
@@ -187,19 +187,19 @@ describe('Viewport', () => {
     });
   });
 
-  it('should update padding when changed only', () => {
+  it.only('should update padding when changed only', () => {
     // Shouldn't call updatePaddingTop since it hasn't changed.
     let bindingMock = sandbox.mock(binding);
     bindingMock.expects('updatePaddingTop').never();
-    viewerViewportHandler();
+    viewerViewportHandler({paddingTop: 19, duration: 200, curve: 'ease-in'});
```

Comment:
With current tests, let's only do `{paddingTop :X}`. Other parameters (duration and curve) we will test explicitly in the follow up PRs. For now we just want to test that this code doesn't break the existing protocol.